### PR TITLE
Update documentation wrt. epoch rewards fractions

### PR DESCRIPTION
**Pull request by rcroessmann**
### Description

Updates the description of the epoch rewards distribution to the carbon offsetting fund, the community fund and the reserve to reflect latest design decisions and discussion around initial values.

### Tested

Not tested.

### Other changes

None

### CODE REVIEW

File: packages/docs/celo-codebase/protocol/proof-of-stake/carbon-offsetting-fund.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 The Carbon Offsetting Fund provides for making the infrastructure of the Celo platform carbon-neutral, by making a transfer every epoch to an organization that commits to using those assets off-chain for carbon offsetting projects.
 
-Through the on-chain governance process, Celo Gold holders can set an on-target carbon offset amount, and the address of a carbon offsetting partner to which to direct these transfers. The on-target amount is adjusted by the epoch [rewards mulitplier](epoch-rewards.md), as with all epoch rewards.
+Through the on-chain governance process, Celo Gold holders can set the fraction of the total desired epoch rewards, initially planned to be 0.5%, that is received by the carbon offset, and the address of a carbon offsetting partner to which to direct these transfers. The on-target amount is adjusted by the epoch [rewards mulitplier](epoch-rewards.md), as with all epoch rewards.
```
Comment (timmoreton) [inline packages/docs/celo-codebase/protocol/proof-of-stake/carbon-offsetting-fund.md]: missing "fund" after "that is received by the carbon offset"

**Review by timmoreton — APPROVED** [review]

File: packages/docs/celo-codebase/protocol/proof-of-stake/carbon-offsetting-fund.md (hunk 1 above)
Comment (nategraf) [reply packages/docs/celo-codebase/protocol/proof-of-stake/carbon-offsetting-fund.md]: An opportunity to use my favorite GitHub feature!
```suggestion
Through the on-chain governance process, Celo Gold holders can set the fraction of the total desired epoch rewards, initially planned to be 0.5%, that is received by the carbon offset fund, and the address of a carbon offsetting partner to which to direct these transfers. The on-target amount is adjusted by the epoch [rewards mulitplier](epoch-rewards.md), as with all epoch rewards.
```

Comment (nategraf) [inline packages/docs/celo-codebase/protocol/proof-of-stake/carbon-offsetting-fund.md]: A link to the governance doc would be helpful.
```suggestion
Through the [on-chain governance process](../governance.md), Celo Gold holders can set the fraction of the total desired epoch rewards, initially planned to be 0.5%, that is received by the carbon offset, and the address of a carbon offsetting partner to which to direct these transfers. The on-target amount is adjusted by the epoch [rewards mulitplier](epoch-rewards.md), as with all epoch rewards.
```

**Review by nategraf — APPROVED** [review]
Left a couple of small suggestions

File: packages/docs/celo-codebase/protocol/proof-of-stake/carbon-offsetting-fund.md (hunk 1 above)
Comment (rcroessmann) [reply packages/docs/celo-codebase/protocol/proof-of-stake/carbon-offsetting-fund.md]: Thanks for catching this! And pretty neat feature indeed.

Comment (rcroessmann) [reply packages/docs/celo-codebase/protocol/proof-of-stake/carbon-offsetting-fund.md]: Agreed and done!

**Pull request merged**